include/he.h

```diff
@@ -608,7 +626,9 @@ typedef enum msg_ids {
   /// Tell the other side that we're closing down
   HE_MSGID_GOODBYE = 12,
   /// Deprecated message - same as Data packet with an unused int flag
-  HE_MSGID_DEPRECATED_13 = 13
+  HE_MSGID_DEPRECATED_13 = 13,
+  /// Contains server configuration data
+  HE_MSGID_SERVER_CONFIG = 14,
```

Comment:
Should this be `HE_MSGID_CLIENT_CONFIG` instead? When I first read it, I thought it would be configuring the server 😊 . Could be call `REMOTE` or something too.

Reply:
I feel the "CLIENT_CONFIG" name is confusing too, because the content of this message is all about the configuration information of the server. And the message flow is server->client only (this is enforced in the code below).

Reply:
Clarified in the comment.